components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -30,6 +30,7 @@
 import org.wso2.carbon.CarbonConstants;
 import org.wso2.carbon.context.CarbonContext;
 import org.wso2.carbon.context.PrivilegedCarbonContext;
+import org.wso2.carbon.core.SameSiteCookie;
```

Comment:
Did we name it as SameSiteCookie ?
IIRC we discussed to make this extended cookie class capable of handling any explicit attributes if needed.
If so, it should have a different naming

Reply:
So the extended cookie seems to be 'ServletCookie' and SameSiteCookie just holds same site modes right.
Then +1